src/main/java/org/pesmypetcare/webservice/controller/petmanager/PetRestController.java

```diff
@@ -99,10 +122,145 @@ public Object getField(@PathVariable String owner, @PathVariable String name,
      * @param field Name of the field to update
      * @param valueMap Entity that contains the value that the field will have. The new field value needs to have the
      *                key "value"
+     * @throws DatabaseAccessException If an error occurs when accessing the database
+     * @throws DocumentException When the document does not exist
+     */
+    @PutMapping("/{owner}/{name}/simple/{field}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void updateSimpleField(@PathVariable String owner, @PathVariable String name, @PathVariable String field,
+                                  @RequestBody Map<String, Object> valueMap)
+        throws DatabaseAccessException, DocumentException {
+        PetEntity.checkSimpleFieldAndValues(field, valueMap.get("value"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "value" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5469160)